install/installer/tools/ambari_setup

```diff
@@ -62,6 +65,9 @@ done
 #Info for log file
 sudo mkdir -p /var/log/trafodion
 sudo chmod 777 /var/log/trafodion
+
+linuxInstalled=$(grep -cr SUSE /proc/version)
```

Comment:
I assume $linuxInstalled is the empty string if it is not SUSE? (Which seems a little odd.) I wonder if a better name for the variable would be $SUSEInstalled?